Makefile

```diff
@@ -1,4 +1,4 @@
-SUPPORTED_COMMANDS := init-module update-module update-module-doc update-module-tf update-module-detectors update-module-outputs check-module
+SUPPORTED_COMMANDS := init-module update-module update-module-doc update-module-tf update-module-detectors update-module-outputs update-severity-doc check-module
```

Comment:
you should not add the target here while we do not want to support any argument for this target